### DISCUSSION TITLE
Problem with getFullList

### DISCUSSION THREAD

**Discussion by MuslemRahimi** [pocketbase/pocketbase#2651] [Q&A] [answered]
Hi,

I have a problem that might be quite trivial but im still wondering if it is a bug

I want to the full list of : 

```
const getNumberOfComments = await locals.pb.collection("comments").getFullList({
        filter: `post="${item.id}"`
        });
```

which gives me an error. However, if i add a "1" everything works fine:

```
const getNumberOfComments = await locals.pb.collection("comments").getFullList(1,{
        filter: `post="${item.id}"`
        });
```

**Comment by ganigeorgiev** [comment, marked as answer]
I'm not able to reproduce it. What version of the JS SDK do you use?

_Note that the object syntax for the `getFullList` call was added in [v0.11.0](https://github.com/pocketbase/js-sdk/releases/tag/v0.11.0) of the JS SDK._

**Reply by MuslemRahimi** [reply]
Sorry my version was still v.0.8. Now it works!

**Comment by dcaponi** [comment]
I'm on 0.15.2 and `const users = await locals.pb.collection('users').getFullList()` returns an empty array for me despite having users according to the UI. I expected without query params to get the full list back. I checked that casing is correct (i.e. all lowercase `users`). 

Am I doing something wrong?

**Reply by ganigeorgiev** [reply]
Make sure that the request satisfy the List/Search API rule of your collection.

**Reply by brynnjocelyn** [reply]
@ganigeorgiev Thank you for this answer! Saved me hours of troubleshooting 😆

**Comment by fahedmahidi** [comment]
A little late but how do you setup your pocketbase ? Through .env file variable ?
An empty array meant for me that the api wasn't actually loaded. You might want to try putting the pocketbase url in full in while instantiating PocketBase in pocketbase.js.
